### DISCUSSION TITLE
Admin UI not accessible anymore after upgrading to rc59

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#5095] [Archive]
Hi all,

I have running rc53 as a dev environment on my nas, with the admin ui page reachable via `http://nas:3000`.

After upgrading to rc59, I can't reach the admin page anymore (app itself is running still fine), the browser is telling me:

> The stylesheet 'http://nas:3000/admin/css/app.ead961e2.css' was not loaded because its MIME type, ”text/html“ is not
> ”text/css"

and 

> The script from “http://nas:3000/admin/js/chunk-vendors.8f8dd28c.js” was loaded even though its MIME type (“text/html”)
> is not a valid JavaScript MIME type.

It seems that it has something to do with `X-Content-Type` but I don't now how to change this without changing the client code...
Or did I miss something while upgrading?

Thanks a lot!

**UPDATE**: If I downgrade to rc53, everything works fine again, without changing something on the server or the apache config, there must be a breaking change that I'm not aware of...

**Comment by rijkvanzanten** [comment]
@sagerio Did you restart the server when upgrading? Sounds like that content type issue might be the API/your webserver throwing 404s for those app files, which could be caused by the fact that an outdated process is trying to fetch the old files on the new distribution 🤔

**Reply by sagerio** [reply]
That brought me to the solution!

Of course, I restarted the server, but: I'm using PM2 to manage the different directus instances, and a simple `pm2 restart myapp` doesn't seem to be sufficient, it worked after `pm2 restart myapp --update-env`.
Maybe useful for future readers ;-)

Thanks a lot!
